src/components/countDown/CountDown.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react'
+import './CountDown.css'
+
+const CountDown = () => {
+  const [day, setDay] = useState('')
+  const [hr, setHr] = useState('')
+  const [min, setMin] = useState('')
+  const [sec, setSec] = useState('')
+
+  let result =
+    Number(day) * 86400 + Number(hr) * 3600 + Number(min) * 60 + Number(sec)
+
+  const inputHandler = (e) => {
+    const id = e.target.id
+    const inputVal = e.target.value
+
+    if (id === 'day') {
+      setDay(inputVal)
+    } else if (id === 'hr' && inputVal < 24 && inputVal >= 0) {
+      setHr(inputVal)
+    } else if (id === 'min' && inputVal < 60 && inputVal >= 0) {
+      setMin(inputVal)
+    } else if (id === 'sec' && inputVal < 60 && inputVal >= 0) {
```

Comment:
I suppose the `<` conditions could be `<=`
I'm not able to enter 24 hours 🤔

Reply:
But 24 hours is a day right, are we allowing users to write n hours too?

Reply:
In `Timer` we have numbers from `0 - 23` only for Hours.
I have checked in my phone.
And for min and sec we have `0 - 59`

Reply:
> I suppose the `<` conditions could be `<=`
> I'm not able to enter 24 hours 🤔

In Timer we have numbers from `0 - 23` only for Hours
And for min and sec we have `0 - 59`
I have checked in my phone

Reply:
> But 24 hours is a day right, are we allowing users to write n hours too?

Yes, we are allowing the user to enter no. of hours.